src/newViews/PillarAnalysis/EntriesFilterForm/index.tsx

```diff
@@ -72,6 +74,20 @@ const commentStatusOptions: KeyValueElement[] = [
     },
 ];
 
+type FormType = FaramValues;
+
+type FormSchema = ObjectSchema<FormType>;
+type FormSchemaFields = ReturnType<FormSchema['fields']>;
+
+const schema: FormSchema = {
+    fields: (): FormSchemaFields => ({
+        created_by: [],
+    }),
```

Comment:
shouldn't we add other fields too?

Reply:
As all fields are dynamic, I am planning to remove this as well.